opentreemap/importer/models/trees.py

```diff
@@ -91,8 +91,6 @@ class TreeImportRow(GenericImportRow):
         'geom': fields.trees.POINT,
         'width': fields.trees.PLOT_WIDTH,
         'length': fields.trees.PLOT_LENGTH,
-        # TODO: READONLY restore when implemented
```

Comment:
The :+1: emoticon on comments is my way of noting that it's been addressed on my host machine.

All the `readonly` comments have been restored. The `readonly` tests that were deleted were in the pending audit tests. They have been restored, but the pending udfs test is skipped with an appropriate message, noted below on the comment about the pending tests.